spec/index.html

```diff
@@ -775,11 +775,14 @@ <h2>Literals and datatypes</h2>
   <p>RDF literals and datatypes are fully described in the
     <a data-cite="RDF12-CONCEPTS#section-Datatypes">Datatypes</a> section of [[!RDF12-CONCEPTS]].
     In summary: with two exceptions, RDF literals combine a string and an IRI <a data-lt="identify">identifying</a> a datatype.
-    The exceptions are <a data-cite="RDF12-CONCEPTS#dfn-language-tagged-string">language-tagged strings</a>, assigned the type <code>rdf:langString</code>,
-    which have two syntactic components, a string and a language tag; and
-    <a data-cite="RDF12-CONCEPTS#dfn-dir-lang-string">directional language-tagged strings</a>,
-    assigned the type <code>rdf:dirLangString</code>,
-    which have three syntactic components, a string, a language tag, and a base direction.
+    The exceptions are <a data-cite="RDF12-CONCEPTS#dfn-language-tagged-string">language-tagged strings</a>,
+    which are assigned the type <code>rdf:langString</code>, and
+    <a data-cite="RDF12-CONCEPTS#dfn-directional-language-tagged string">directional language-tagged strings</a>,
+    which are assigned the type <code>rdf:dirLangString</code>.
+    <a data-cite="RDF12-CONCEPTS#dfn-language-tagged-string">Language-tagged strings</a>
+    have two syntactic components, these being a string and a language tag;
+    <a data-cite="RDF12-CONCEPTS#dfn-directional-language-tagged string">directional language-tagged strings</a>
+    have three syntactic components, these being a string, a language tag, and a base direction.
```

Comment:
Granting @doerthe's request to change the gerund constructs to colons can be done by this refactoring...
```suggestion
    The exceptions are
    <a data-cite="RDF12-CONCEPTS#dfn-language-tagged-string">language-tagged strings</a>
    and
    <a data-cite="RDF12-CONCEPTS#dfn-directional-language-tagged string">directional language-tagged strings</a>.
    <a data-cite="RDF12-CONCEPTS#dfn-language-tagged-string">Language-tagged strings</a>
    are assigned the type <code>rdf:langString</code>, and
    have two syntactic components: a string, and a language tag.
    <a data-cite="RDF12-CONCEPTS#dfn-directional-language-tagged string">Directional language-tagged strings</a>
    are assigned the type <code>rdf:dirLangString</code>, and
    have three syntactic components: a string, a language tag, and a base direction.
```

Reply:
... or this one...
```suggestion
    The exceptions are <a data-cite="RDF12-CONCEPTS#dfn-language-tagged-string">language-tagged strings</a>,
    which are assigned the type <code>rdf:langString</code>, and
    <a data-cite="RDF12-CONCEPTS#dfn-directional-language-tagged string">directional language-tagged strings</a>,
    which are assigned the type <code>rdf:dirLangString</code>.
    <a data-cite="RDF12-CONCEPTS#dfn-language-tagged-string">Language-tagged strings</a>
    have two syntactic components: a string, and a language tag;
    <a data-cite="RDF12-CONCEPTS#dfn-directional-language-tagged string">directional language-tagged strings</a>
    have three syntactic components: a string, a language tag, and a base direction.
```

Reply:
There is no assignment here, so "assigned" is wrong.

Reply:
Since no-one else has commented, I think this can be merged once @pfps (and then @doerthe) are satisfied with this change.

@pfps — Perhaps you could suggest a better word than "assigned"? Or maybe rephrasing like this would work for you?
```suggestion
    The exceptions are <a data-cite="RDF12-CONCEPTS#dfn-language-tagged-string">language-tagged strings</a>,
    which have the type <code>rdf:langString</code>, and
    <a data-cite="RDF12-CONCEPTS#dfn-directional-language-tagged string">directional language-tagged strings</a>,
    which have the type <code>rdf:dirLangString</code>.
    <a data-cite="RDF12-CONCEPTS#dfn-language-tagged-string">Language-tagged strings</a>
    have two syntactic components: a string, and a language tag;
    <a data-cite="RDF12-CONCEPTS#dfn-directional-language-tagged string">directional language-tagged strings</a>
    have three syntactic components: a string, a language tag, and a base direction.
```

Reply:
The change above is a decided improvement.  It appears that I can commit it to the PR, so I'm going to do so.